lib/ui/viewer/file/file_caption_widget.dart

```diff
@@ -13,7 +13,7 @@ class FileCaptionWidget extends StatefulWidget {
 }
 
 class _FileCaptionWidgetState extends State<FileCaptionWidget> {
-  int maxLength = 280;
```

Comment:
We also need to ellipsize on the photo screen, will that also happen with this change? Maybe that description widget on the photo screen have a max height already...